cli/azd/test/functional/cli_test.go

```diff
@@ -201,6 +202,9 @@ func Test_CLI_InfraCreateAndDeleteWebApp(t *testing.T) {
 	res, err := http.Get(url)
 	require.NoError(t, err)
 
+	_, err = executil.RunCommandWithShell(ctx, "dotnet", "user-secrets", "list", "--project", path.Join(dir, "/src/dotnet/webapp.csproj"))
```

Comment:
This got inserted in the middle of the test, could you move L205-L207 to L213 (after the assert about ```"Hello `azd`."```)

Let's also validate here that what we got back from user-secrets list has the `WEBSITE_URL` secret set.

Reply:
Use `filepath.Join` instead of `path.Join`